api/language-extensions/syntax-highlight-guide.md

```diff
@@ -325,4 +315,209 @@ If you can't add a `meta.embedded.*` scope to your grammar, you can alternativel
 }
 ```
 
+## Semantic Tokenization
```

Comment:
As suggested, let's move this to a new topic.
I suggest `https://code.visualstudio.com/api/language-extensions/semantic-highlight-guide`